api/docker/snapshot.go

```diff
@@ -64,6 +69,23 @@ func snapshotInfo(snapshot *portainer.Snapshot, cli *client.Client) error {
 	return nil
 }
 
+func snapshotNodes(snapshot *portainer.Snapshot, cli *client.Client) error {
+	nodes, err := cli.NodeList(context.Background(), types.NodeListOptions{})
+	if err != nil {
+		return err
+	}
+	var nanoCpus int64
+	var totalMem int64
+	for _, node := range nodes {
+		resources := node.Description.Resources
+		nanoCpus += resources.NanoCPUs
+		totalMem += resources.MemoryBytes
+	}
+	snapshot.TotalCPU = int(nanoCpus / 1e9)
```

Comment:
I believe that this information should be processed in the frontend rather than applying the `/ 1e9` in the backend. It probably needs an Angular filter.

Reply:
In other places where it isn't a swarm, you're expecting the right number, without a filter, no?
I think it should be consistent

Reply:
Yup, and in that case we can simply re-use that filter as well. I'd like to stay as close as possible of the results of the Docker API and apply the processing layer in the frontend.